lib/alice/handlers/karma.ex

```diff
@@ -46,11 +53,15 @@ defmodule Alice.Handlers.Karma do
     |> reply("#{term} has had its karma scattered to the winds. (#{count})")
   end
 
+  defp _increment(term, term, conn), do: reply(conn, ~s(#{@karma_denial_message}))
```

Comment:
You don't need the underscore for private methods. Just the fact that it's a different arity is good enough.

Reply:
Makes sense. I mainly did it to be explicit about the helper function.